examples/synthetics_mesh_example_test.go

```diff
@@ -14,157 +13,21 @@ import (
-		fmt.Println("Num health items:", len(healthItems))
-		if len(healthItems) > 0 {
-			return healthItems[0].Mesh, nil
-		} else {
-			return nil, nil
-		}
-	} else {
-		fmt.Println("[no health items received]")
-		return nil, nil
-	}
-}
-
-func printMetricsMatrix(matrix metricsMatrix) {
-	w := makeTabWriter()
-
-	// print table header
-	header := ".\t"
-	for _, x := range matrix.agents {
-		header = header + x + "\t"
-	}
-	fmt.Fprintln(w, header)
-
-	// print table rows
-	for _, fromAgent := range matrix.agents {
-		row := fromAgent + "\t"
-		for _, toAgent := range matrix.agents {
-			if metrics, ok := matrix.getMetric(fromAgent, toAgent); ok {
-				row = row + formatLatency(metrics) + "\t"
-			} else {
-				row = row + "[X]\t"
-			}
-		}
-		fmt.Fprintln(w, row)
-	}
-
-	w.Flush()
-}
-
-func makeTabWriter() *tabwriter.Writer {
-	const minWidth = 0  // minimal cell width including any padding
-	const tabWidth = 2  // width of tab characters (equivalent number of spaces)
-	const padding = 4   // distance between cells
-	const padchar = ' ' // ASCII char used for padding
-	const flags = 0     // formatting control
-	w := tabwriter.NewWriter(os.Stdout, minWidth, tabWidth, padding, padchar, flags)
-	return w
-}
-
-func formatLatency(metrics *synthetics.V202101beta1MeshMetrics) string {
-	latency, err := strconv.ParseInt(*metrics.GetLatency().Value, 10, 64)
-	if err != nil {
-		return "error"
-	}
-
-	return strconv.FormatInt(latency/1000, 10) + "ms" // latency is returned in thousands of milliseconds, so need to divide by 1000
-}
-
-// metricsMatrix holds "fromAgent" -> "toAgent" connection metrics
-type metricsMatrix struct {
-	agents []string
-	cells  map[string]map[string]*synthetics.V202101beta1MeshMetrics
-}
-
-func newMetricsMatrix(mesh []synthetics.V202101beta1MeshResponse) metricsMatrix {
-	// fill agents
-	agents := []string{}
-	for _, agent := range mesh {
-		agents = append(agents, agent.GetAlias())
-	}
-
-	// fill matrix cells
-	cells := make(map[string]map[string]*synthetics.V202101beta1MeshMetrics)
-	for _, fromAgent := range mesh {
-		cells[fromAgent.GetAlias()] = make(map[string]*synthetics.V202101beta1MeshMetrics)
-		for _, toAgent := range *fromAgent.Columns {
-			cells[fromAgent.GetAlias()][toAgent.GetAlias()] = toAgent.Metrics
-		}
-	}
-	return metricsMatrix{agents: agents, cells: cells}
-}
-
-func (m metricsMatrix) getMetric(fromAgent string, toAgent string) (*synthetics.V202101beta1MeshMetrics, bool) {
-	toAgents, ok := m.cells[fromAgent]
-	if !ok {
-		return nil, false
-	}
-
-	metric, ok := toAgents[toAgent]
-	if !ok {
-		return nil, false
-	}
-
-	return metric, true
-}
-
-func TestGetMeshTestResultsGRPCExample(t *testing.T) {
 	assert.NoError(t, runGetMeshTestResultsGRPC())
 }
 
 func runGetMeshTestResultsGRPC() error {
-	flag.Parse()
+	testID, err := pickTestID()
+	if err != nil {
+		return err
+	}
 
-	mesh, err := getMeshTestResultsGRPC(*testID)
+	mesh, err := getMeshTestResultsGRPC(testID)
```

Comment:
-> `getMeshTestResults`
There is no need for gRPC suffix. In other places in examples/tests too.